### PR TITLE
EMERG PATCH: fix default readability bug that keeps bright red/blue a…

### PR DESCRIPTION
**Pull request by aryanrsuri**
…s deault if cookie is null

### CODE REVIEW

File: public/Aryan Suri/Sidebar/sidebar.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 "use strict";
 var CONVERSION_CALCULATOR;
+if (localStorage.getItem("beeline") === null) {
```
[the remark this answers is not in the document]
Comment (Miniland1333) [reply public/Aryan Suri/Sidebar/sidebar.js:3]: Also, why is beeline localStorage but the other readability items are sessionStorage?

File: public/Aryan Suri/Sidebar/sidebar.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
 "use strict";
 var CONVERSION_CALCULATOR;
-
-
+if (localStorage.getItem("beeline") === null) {
```
Comment (Miniland1333) [inline public/Aryan Suri/Sidebar/sidebar.ts:3]: Same comments as the javascript file

**Review by Miniland1333 — DISMISSED** [review]
I have a quick code suggestion and I have a question inline about localStorage vs sessionStorage.

File: public/Aryan Suri/Sidebar/sidebar.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (aryanrsuri) [reply public/Aryan Suri/Sidebar/sidebar.js:3]: im not sure, it had something to do with importance, but next patch i will put them all under localStorage

**Review by Miniland1333 — APPROVED** [review]
Changes look good and might as well have both checks.
-Henry

**Pull request merged**